kuma/wiki/tests/test_views.py

```diff
@@ -339,17 +339,23 @@ def test_footer_contributors(self, get_contributors, flag_is_active):
         # just checking if the contributor link is rendered
         eq_(len(contributors.find('a')), 2)
 
-    def test_revision_view_bleached_content(self):
-        """Bug 821988: Revision content should be cleaned with bleach"""
-        d, r = doc_rev("""
-            <a href="#" onload=alert(3)>Hahaha</a>
-            <svg><svg onload=alert(3);>
-        """)
+    def test_template_revision_content(self):
+        doc = document(title='Testing Template', slug='Template:Testing', save=True)
+        r = revision(save=True, document=doc, is_approved=True)
+
         resp = self.client.get(r.get_absolute_url())
-        page = pq(resp.content)
-        ct = page.find('#wikiArticle').html()
-        ok_('<svg>' not in ct)
-        ok_('<a href="#">Hahaha</a>' in ct)
```

Comment:
Bleach is not being used for quite some time, [here](https://github.com/mozilla/kuma/commit/635b2bf59b2b6ecb93990006546261d530c58aed).

I assumed since bleach was replaced for "content_cleaned" it already had a test.
After a quick look at test_models.py I didn't found any reference for "content_cleaned".

Maybe I should create a new test for this at the test_models package or just restore the view test and reword it to make clear that it's testing that the "content_cleaned" is being used. What do you think?